smartcontractkit/external-adapters-js
# Feature/DF-19166 build kaiko ea for realized vol

## Closes #[DF-19166](https://smartcontract-it.atlassian.net/browse/DF-19166)

## Description
Add realized volatility endpoint to Kaiko EA

## Changes

- Added realized-vol endpoint to Kaiko EA according to spec
- Related tests
- Tiny refactor to standardize base Kaiko API across endpoints

## Steps to Test

1. Run Kaiko EA with `yarn start`
2. Query `realized-vol` endpoint with standard payload to sanity check connection with Kaiko & returned data
3. Regression test default `price` (`trades`) endpoint

## Quality Assurance



[DF-19166]: https://smartcontract-it.atlassian.net/browse/DF-19166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
